library/std/src/sys/unix/process/process_unix.rs

```diff
@@ -552,6 +552,33 @@ impl Command {
             Ok(Some(p))
         }
     }
+
+    #[cfg(target_os = "freebsd")]
+    #[allow(dead_code)]
+    pub fn sandbox(&mut self) -> bool {
+        let r = unsafe { libc::cap_enter() };
+        r == 0
+    }
```

Comment:
One other thing I noticed is that I don't think this is correct. [The syscall docs](https://www.cl.cam.ac.uk/research/security/capsicum/man/cap_enter.2.pdf) say that this sandboxes the current process and subsequent children, but given this is a _child_ process _builder_, I don't think anyone would expect calling this to change the the current process. Thats also not how every sandboxing architecture works, because some have the spawner use its greater permissions to constrain untrusted children (Chromium's `setuid` sandbox, to give an example).

So, I think this should be done post-fork and pre-exec during the child proc spawning operation so that the spawner is not affected.